docs/README.md

```diff
@@ -16,4 +16,7 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
-This is the public documentation site for Superset, built using [Docusaurus 2](https://docusaurus.io/). See [CONTRIBUTING.md](../CONTRIBUTING.md#documentation) for documentation on contributing to documentation.
+This is the public documentation site for Superset, built using
```

Comment:
On a recent PR I was advised by @john-bodley not to add manual line breaks in the docs: https://github.com/apache/superset/pull/26038#discussion_r1401314027 .  It seems like these are added manual breaks, and then the new docs page has manual breaks?

I don't see any guidance either way in our style guide or contributing guidelines.

Reply:
Actually I don't think i touched this did I? Personally I like line breaks and happily will get into a tabs-vs-space religious debate over it :) Main point is around "diffing". If you have a long paragraph and fix a typo, you're in the `git blame` for that whole line, and diffs will be much more verbose than needed. I vote for line breaks soft at 80-90, hard at 100, and across file most file types.

Reply:
Great, I am agnostic about which way to go but agree it should not hold up this PR.  We can talk about it at a townhall or something and get on the same page there.